assets/src/dashboard/app/views/templateDetails/components.js

```diff
@@ -44,58 +45,7 @@ export const DetailContainer = styled.section.attrs({
 `;
 
 export const Column = styled.div`
-  ${({ theme }) => `
```

Comment:
To _try_ (and fail really) to keep the diff easier to read I only moved styled components out of this `templateDetails/components` when they were unnecessary or had a new base - like using design system stuff. Then I moved them more locally. This is a bit of a mixed bag and I'll clean it up better outside of these updates.